agent/api/ecsclient/client_test.go

```diff
@@ -231,13 +235,34 @@ func TestSubmitContainerStateChangeLongReason(t *testing.T) {
 	}
 }
 
-func TestRegisterContainerInstance(t *testing.T) {
+func buildAttributeList(capabilities []string, attributes map[string]string) []*ecs.Attribute {
+	var rv []*ecs.Attribute
+	for i := range capabilities {
+		rv = append(rv, &ecs.Attribute{Name: aws.String(capabilities[i])})
+	}
+	for key, value := range attributes {
+		rv = append(rv, &ecs.Attribute{Name: aws.String(key), Value: aws.String(value)})
+	}
+	return rv
+}
+
+func TestReRegisterContainerInstance(t *testing.T) {
+	additionalAttributes := map[string]string{"my_custom_attribute": "Custom_Value1",
```

Comment:
Is it possible some attributes just have name without value? Like:
```
{"name": "com.amazonaws.ecs.capability.logging-driver.json-file"}
```
Just want to cover that in the test.